examples/Demo/Shared/App.razor

```diff
@@ -1,11 +1,11 @@
 ﻿<Router AppAssembly="@typeof(App).Assembly">
     <Found Context="routeData">
-        <RouteView RouteData="@routeData" DefaultLayout="@typeof(DemoMainLayout)" />
+        <RouteView RouteData="@(routeData)" DefaultLayout="@typeof(DemoMainLayout)" />
```

Comment:
Why do you change this code?

Reply:
Nothing Changed there, I had some test code there and removed it, seems like format gets messed up, I'll fix that